arangod/Transaction/Methods.cpp

```diff
@@ -3944,8 +3941,7 @@ Future<OperationResult> Methods::updateInternal(
     f = modifyCoordinator(collectionName, newValue, options,
                           TRI_VOC_DOCUMENT_OPERATION_UPDATE, api);
   } else {
-    OperationOptions optionsCopy = options;
-    f = modifyLocal(collectionName, newValue, optionsCopy, /*isUpdate*/ true);
+    f = modifyLocal(collectionName, newValue, options, /*isUpdate*/ true);
```

Comment:
nit: is it possible to std::move the `options` here into `modifyLocal(...)`? That would safe a copy of the options object, which contains strings with heap-allocations attached to them.